src/AltinnCore/Common/Services/Implementation/DataAppSI.cs

```diff
@@ -90,7 +91,7 @@ public async Task<Instance> InsertData<T>(T dataToSerialize, Guid instanceGuid,
         }
 
         /// <inheritdoc />
-        public void UpdateData<T>(T dataToSerialize, Guid instanceGuid, Type type, string org, string app, int instanceOwnerId, Guid dataId)
+        public async Task<Instance> UpdateData<T>(T dataToSerialize, Guid instanceGuid, Type type, string org, string app, int instanceOwnerId, Guid dataId)
```

Comment:
Mulig det er meg, men er det ikke rart å kalle denne updateDAta når den leser data?

Reply:
It also returns what was stored

Reply:
I realize the problem. i did not see there was some lines missing in the view.